bcos-tool/bcos-tool/NodeConfig.cpp

```diff
@@ -56,7 +57,12 @@ NodeConfig::NodeConfig(KeyFactory::Ptr _keyFactory)
 
 void NodeConfig::loadConfig(boost::property_tree::ptree const& _pt, bool _enforceMemberID)
 {
-    loadChainConfig(_pt);
+    // if version < 3.1.0, config.ini include chianConfig
+    if (m_compatibilityVersion < (uint32_t)bcos::protocol::Version::V3_1_VERSION &&
```

Comment:
Please make sure that loadConfig() is run after loadGenesisConfig()
For example, in ExecutorServiceApp.cpp:140, loadGenesisConfig is running after loadConfig, so we can't get m_compatibilityVersion in loadConfig

Reply:
storageTool.cpp
AirNodeInitializer.cpp
SchedulerServiceApp.cpp
Initializer.cpp
lightnode/main.cpp

Reply:
Please add ut to make sure that logic